resume/forms.py

```diff
@@ -132,10 +132,9 @@ class resumebuilder(FlaskForm):
     designation = StringField("Designation",
         validators=[DataRequired(),Length(min=3)])
     email = StringField("Email Id",
-        validators=[DataRequired(),Length(min=5)])
+        validators=[Email(),DataRequired()])
     phoneno= StringField("Phone No",
-        validators=[DataRequired(),Length(min=5)])
-
+        validators=[Regexp('^[0-9]*$')])
```

Comment:
hey this looks correct , but since you removed the datarequired validator I am able to submit that part of the form without entering a phone no , can you fix this please .
Rest seems to working good , Ill merge them after that then

Reply:
Done!